bash/solutii/cristina_ungureanu/compare.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [[ `diff -r $1 $2 | wc -c` -gt 0 ]]
```

Comment:
In loc de '` ' poti folosi `$(comanda)`. Iti recomand sa te folosesti de exit_codul din `diff` pentru a face testul.